easybuild/easyconfigs/m/magma/magma-2.5.1-CrayIntel-19.10-cuda-10.1.eb

```diff
@@ -0,0 +1,37 @@
+# @author: wsawyer
+
+easyblock = 'MakeCp'
+
+name = 'magma'
+version = '2.5.1'
+versionsuffix = '-cuda-10.1'
+
+homepage = 'http://icl.cs.utk.edu/magma/'
+description = """The MAGMA project aims to develop a dense linear algebra
+library similar to LAPACK but for heterogeneous/hybrid architectures, starting
+with current Multicore+GPU systems."""
+
+toolchain = {'name': 'CrayIntel', 'version': '19.10'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://icl.cs.utk.edu/projectsfiles/magma/downloads/']
+patches = [('make.inc-2.5.1', './')]
+#dependencies = [('craype-accel-nvidia60', EXTERNAL_MODULE)]
+
+builddependencies = [
+  ('cudatoolkit', EXTERNAL_MODULE)
```

Comment:
We suggest to pin the cuda version (see comments in #1404 )

Reply:
done